src/Common/src/Interop/Windows/mincore/Interop.CopyFile.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        internal static int CopyFile(string src, string dst, bool failIfExists)
+        {
+            int copyFlags = failIfExists ? Interop.mincore.FileOperations.COPY_FILE_FAIL_IF_EXISTS : 0;
+            int cancel = 0;
+            if (!Interop.mincore.CopyFileEx(src, dst, IntPtr.Zero, IntPtr.Zero, ref cancel, copyFlags))
+            {
+                return Marshal.GetLastWin32Error();
+            }
+
+            return Interop.mincore.Errors.ERROR_SUCCESS;
```

Comment:
I'm not understanding the benefit of returning the error like this, rather than just returning the Boolean result and letting the caller do with it as they will (as we do with other interop).  Case in point, it looks like there's a single caller of this method, and it only cares whether it succeeded or not.